pkg/models/const.go

```diff
@@ -15,7 +15,7 @@ const (
 )
 
 var (
-	PassThroughHosts = []string{"dc.services.visualstudio.com"}
+	PassThroughHosts = []string{"^dc\\.services\\.visualstudio\\.com$"}
```

Comment:
Can you explain this change.?

Reply:
without this we get a security error as regex of this url would be matched with other urls, got a error regarding the same in pipeline